app/classifier/tasks/survey/chooser.cjsx

```diff
@@ -11,6 +16,27 @@ module.exports = React.createClass
     onFilter: Function.prototype
     onChoose: Function.prototype
 
+  componentWillMount: ->
+    # refs for the choices
+    @choiceButtons = []
```

Comment:
Yes, this is a workaround for #3564, since the buttons aren't rendered in the same order that they display visually.

`@choiceButtons` is recalculated after a render, so I kept it out of component state.